android/engine/src/main/assets/configs/default/config_composition.json

```diff
@@ -90,6 +90,26 @@
         "reference": "Binary/56181",
         "identifier": { "value": "sync" }
       }
+    },
+    {
+      "title": "Add Family",
+      "focus": {
+        "reference": "Questionnaire/82952",
+        "identifier": {
+          "value": "family-registration"
+        }
+      },
+      "mode": "working"
+    },
+    {
+      "title": "Add Family Member",
+      "focus": {
+        "reference": "Questionnaire/82953",
+        "identifier": {
+          "value": "family-member-registration"
+        }
+      },
+      "mode": "working"
```

Comment:
@roywanyaga This composition should be added to the eCBIS folder here https://github.com/opensrp/fhir-resources. 

Why are the StructureMaps used by the Questionnaires missing from the composition?

Reply:
[pull request to update the composition and other debug resources for eCBIS](https://github.com/opensrp/fhir-resources/pull/65) has been created: